### PR TITLE
Feature/scroll to top 

### LINKED ISSUE #54: [Feature] Add a scroll-to-top element to UX for longer page contexts
## Description

Many pages within Creative Commons websites have much longer pages, in these cases a `scroll-to-top` element would serve a valid purpose in certain contexts to aid in improved UX.

The element would likely not be present on smaller screens which already have some measure of functionality related to this baked into their operating system level UX; but for larger viewport sizes this could be much more helpful to include.

## Additional context
* https://github.com/creativecommons/cc-legal-tools-app/issues/474
* https://github.com/creativecommons/cc-resource-archive/pull/216

Solutions here should explore non-js interventions. CSS pseudo elements using z-index stack positions and variable percentage based heights would offer a means to "reveal" the button after a degree of scrolling, and to visually hide the element on pages which do not reach a particular length. Special care for keyboard accessibility should be taken under consideration to ensure that it can receive appropriate focus and order handling.

## Implementation

- [x] I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by Daodu111**
## Fixes

- Fixes #54 by @TimidRobot 

## Description

This pull request adds a scroll-to-top element to enhance user experience on longer pages. The new feature allows users to easily navigate back to the top of the page with a single click.

## Technical details

I implemented a scroll-to-top button using HTML and CSS only and it appears after scrolling down a little distance

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main` or `master`).
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (possumbilities) [inline .vscode/settings.json:3]: this file shouldn't have been commited

File: docs/index.html [hunk 2]
```diff
@@ -970,6 +969,7 @@ <h1>Title Here</h1>
 </pre>
 
 
+
```
Comment (possumbilities) [inline docs/index.html:972]: why the extra space?

File: .vscode/settings.json (hunk 1 above)
Comment (Daodu111) [reply .vscode/settings.json:3]: I’ll remove it and create a PR again. Hope that would help and also I want to ask if the function works as requested?

File: docs/index.html (hunk 2 above)
Comment (Daodu111) [reply docs/index.html:972]: I’ll also work on this as soon as possible

**Comment by possumbilities** [comment]
Hi @Daodu111 Thank you for the contribution! After looking at it further, I'm not sure that this meets the criteria outlined in the Issue. 

For example: 

> The element would likely not be present on smaller screens which already have some measure of functionality related to this baked into their operating system level UX; but for larger viewport sizes this could be much more helpful to include.

Additionally, this does not utilize CSS pseudo elements like `:before` or `:after`

It also includes a new icon outside the standard icon library implementation, and should follow how icons are included within the framework already. 

And I believe you need to include the scroll-to-top on an existing specimen or create a new one for display/testing purposes. One could argue adding it to the `archive-page` context might suffice since it has sufficient height, has two columns, and might benefit from such a button. A longer context like `blog-post` might work as well or in addition.


I think the approach needs to be changed in several areas before it could proceed. I'll leave it open for now to leave room for you to adjust it.

File: specimen/index-logo.html [hunk 3]
```diff
@@ -18,12 +18,12 @@ <h1><a class="identity-logo" href="#">Creative Commons</a></h1>
         <button class="expand-menu">Menu</button>
         <nav class="primary-menu">
             <ul>
-                <li><a href="#">Who We Are</a></li>
-                <li><a href="#">What We Do</a></li>
-                <li><a href="#">Licenses and Tools</a></li>
-                <li><a href="#">Blog</a></li>
-                <li><a href="#">Support Us</a></li>
-                <li><a class="attention" href="https://summit.creativecommons.org/">Global Summit 2023</a></li>
+                <li><a href="#" class="active">Who We Are</a></li>
```
Comment (possumbilities) [inline specimen/index-logo.html]: these changes are not related to this PR.
